design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
```

Comment:
@baijum sorry for the confusion. I hope the BJNs call did clear things up. I will quickly make comments in the PR as well to reflect my suggestions.